Cargo.toml

```diff
@@ -8,3 +8,6 @@ members = [
 
 [profile.release]
 debug=true
+
+[patch.crates-io]
+nom-sql = { path = "../nom-sql" }
```

Comment:
This is probably not something we want to commit :)

Reply:
Yup! I'll do that last; I want it to still work while I'm testing it :)